src/Unity.Mathematics.CodeGen/VectorGenerator.cs

```diff
@@ -192,6 +192,17 @@ private static void WriteFile(string filename, string text)
             // Generate auto generated comment
             text = s_AutoGenHeader + text;
 
+            // Trim trailing spaces that could have come from code gen.
+            char[] trim = { ' ' };
```

Comment:
You might want to add tab too there, just in case...

Reply:
I got you fam!

<img width="846" alt="image" src="https://user-images.githubusercontent.com/3121968/74989597-b0693800-53f5-11ea-8706-09a9c889e0a2.png">